code/modules/client/preferences_toggle_procs.dm

```diff
@@ -382,7 +382,7 @@
 /client/verb/toggle_status_indicators()
 	set name = "Toggle Status Indicators"
 	set category = "Preferences"
-	set desc = "Enable/Disable seeing status indicators over peoples' heads."
+	set desc = "Toggles seeing status indicators over people's heads."
```

Comment:
`Peoples'` is correct here, because it refers to status indicators belonging to people. Plural possessives are fucky, welcome to English.